src/layout/verticalPanel/statusArea.ts

```diff
@@ -197,7 +202,12 @@ export class MsStatusArea extends Clutter.Actor {
         // All icons actors should have a single child.
         const mainChild = actor.get_children()[0] as Clutter.Actor | undefined;
         if (mainChild !== undefined) {
-            if (mainChild instanceof (compareVersions(gnomeVersionNumber, parseVersion('43.0')) >= 0 ? panel.QuickSettings : panel.AggregateMenu)) {
+            if (
+                mainChild instanceof
+                (compareVersions(gnomeVersionNumber, parseVersion('43.0')) >= 0
```

Comment:
I'd prefer this check to be stored in a const variable.

Reply:
done